src/main/java/org/medicmobile/webapp/mobile/ConnectionErrorActivity.java

```diff
@@ -40,7 +41,12 @@ public class ConnectionErrorActivity extends ClosableAppActivity {
 				}
 			}
 		};
-		registerReceiver(broadcastReceiver, new IntentFilter("onPageFinished"));
+		ContextCompat.registerReceiver(
```

Comment:
These were the main changes needed for Android 14 compatibility (and were the source of the errors we have seen in the field).  https://developer.android.com/about/versions/14/behavior-changes-14#runtime-receivers-exported